packages/manager/core/vite-config/src/config.js

```diff
@@ -12,8 +13,10 @@ const runInContainer = process.env.CONTAINER;
 
 const getBaseConfig = (config) => {
   const envConfig = config || {};
-  if (envConfig.isLABEU) {
-    envConfig.host = 'www.build-ovh.com';
+  dotenv.config({ path: resolve(process.cwd(), '../../../../.env') });
```

Comment:
Can you add a try/catch in case the file doesn't exist to throw an explicit error that says "create the .env file please" ?

Reply:
good idea, fixed